load.php

```diff
@@ -291,13 +300,23 @@ function perflab_is_standalone_plugin_loaded( $module ) {
  * Gets the standalone plugin constants used for each module / plugin.
  *
  * @since 2.2.0
+ * @since n.e.x.t The `$type` parameter was added.
  *
- * @return array Map of module path to version constant used.
+ * @param string $type Optional. Either 'modules' or 'plugins'. Default 'modules'.
+ * @return array Map of module path / plugin slug and the version constant used.
  */
-function perflab_get_standalone_plugins_constants() {
+function perflab_get_standalone_plugins_constants( $type = 'modules' ) {
```

Comment:
It seems a bit strange that this function is named to get the "standalone plugins" but the default `$type` is not to get the standalone plugins at all, but rather the modules. Why not create a separate function like `perflab_get_module_constants()` instead of adding a `$type` arg?

Reply:
Happy to do that if y'all agree, but that's a BC break, so I went the "safe" route first.

It's probably fair to assume nobody uses these functions outside of PL, but if they did this would break. Let me know what you prefer.

Reply:
Yeah, there's [no other plugins](https://wpdirectory.net/search/01HMSNB2Q7PCE5HNDAJH450NB2) using this function. So I think it's fine to break back-compat. 

Otherwise, two new functions could be created to include `version`:

* `perflab_get_standalone_plugin_version_constants()`
* `perflab_get_module_version_constants()`

The existing function can then be marked as deprecated. But this is probably overkill.

Reply:
SG regarding backward compatibility.

However those two names wouldn't be clear either. To clarify:
* Everything listed here is only relevant for standalone plugins.
* The first list is for modules that have an equivalent standalone plugin available.
* The second list is for standalone plugins overall, regardless of whether they come from a module or are _exclusively_ available as a standalone plugin.

Reply:
I've updated this in https://github.com/WordPress/performance/pull/949/commits/8181c68ae8806ecd89c047d76318044ae24b01f8.

For now I just deprecated the other function. Call it me being overly cautious because of WP core. If you prefer to drop it, we can, but I don't really see a value in immediately removing it.